### PR TITLE
bugzilla: announce PR states when evaluating post-merge

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Previously, when we handled a PR merging we'd check the linked bug for
PRs and move it if all of them were merged, but we were light on details
for which PRs we'd checked and we posted nothing if there was nothing to
do. This change updates our messaging to be more explicit and posts an
explanation fo why we did nothing, when we do nothing.

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @wking

### CODE REVIEW

File: prow/plugins/bugzilla/bugzilla_test.go [hunk 1]
```diff
@@ -923,9 +927,22 @@ Instructions for interacting with me using PR comments are available [here](http
...
 			expectedBug: &bugzilla.Bug{ID: 123},
+			expectedComment: `org/repo#1:@user: Some pull requests linked via external trackers have merged: [org/repo#1](https://github.com/org/repo/pull/1). The following pull requests linked via external trackers have not merged:
+ * [org/repo#22](https://github.com/org/repo/pull/22) is open
+[Bugzilla bug 123](www.bugzilla/show_bug.cgi?id=123) has been moved to the MODIFIED state.
+
+<details>
+
+In response to [this](http.com):
+
+>Bug 123: fixed it!
```
Comment (wking) [inline prow/plugins/bugzilla/bugzilla_test.go]: I'd expect `> Bug...` (with a space) here, but this seems to be pre-existing code, so feel free to ignore.

**Comment by wking** [comment]
[Bazel][1] grumps about gofmt.

[1]: https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/test-infra/16944/pull-test-infra-bazel/1242842761980284929

**Comment by stevekuznetsov** [comment]
I will spend time investigating why my editor, which is configured to `gofmt`, messes that up. :angry:

**Comment by stevekuznetsov** [comment]
Without a space:

>Bug

With a space:

> Buig

**Pull request merged**